clang/include/clang/Basic/AttrDocs.td

```diff
@@ -3492,6 +3492,42 @@ conjunction with disable_loop_pipelining.
   }];
 }
 
+def SYCLIntelEnableLoopPipeliningAttrDocs : Documentation {
+  let Category = DocCatVariable;
+  let Heading = "intel::enable_loop_pipelining";
+  let Content = [{
+The ``enable_loop_pipelining`` attribute applies to a loop in SYCL device code.
+Takes no arguments and enables pipelining of the loop. This attribute is useful
+in the low-area flow, in which all loops are unpipelined by default. Cannot be
+used on the same loop in conjunction with ``disable_loop_pipelining`` attribute.
```

Comment:
Can you also update the documentation for ``disable_loop_pipelining`` to add that it can't be used with ``enable_loop_pipelining``?

Reply:
Thank you @premanandrao for reviews and feedbacks. 

>>Can you also update the documentation for disable_loop_pipelining to add that it can't be used with enable_loop_pipelining?

Yes, i have updated documentation.